### DISCUSSION TITLE
Self host

### DISCUSSION THREAD

**Discussion by Fishhyyy** [sleeyax/aternos-discord-bot#36] [Q&A] [answered]
Is there any way you can show me or send me a video on how to host the bot
Fishy#9556

**Comment by AyushSainiIndian** [comment]
pls send  a copy of the video to me also 
AyushSainipro1#5995

**Comment by sleeyax** [comment, marked as answer]
**If** I make a video it will be on YouTube.

**Reply by AyushSainiIndian** [reply]
then make it we have no problem 
if u allow me i can also make one video but i dont know how to self host

**Reply by sleeyax** [reply]
> then make it we have no problem

It takes time to prepare, make and edit a video like that. There's users on Windows, Mac and Linux, there's different kind of builds and hosting providers, db set up is optional etc. A mix of possibilities which makes it difficult to put all of it in a video format.

>  but i dont know how to self host

It's literally in  [the README](https://github.com/sleeyax/aternos-discord-bot#self-hosting)... If you can't wait, *why not take the time to read that instead*?

**Reply by Rockey7164** [reply]
Your README is hella confusing , listen if more then 2 person are saying that they are unable to understand from your README it means that README isnt doing what its supposed to do . i say the best way is Make a README seapertely only FOR windows with proper step by step instruction from 0 to 10 where a kindergardener could understand,That will definetly reduce the amount of people asking for VIDEO's

**Reply by sleeyax** [reply]
> Your README is hella confusing 

What's confusing about it? If no one tells me I can't improve  ¯\_(ツ)_/¯

>  i say the best way is Make a README seapertely only FOR windows with proper step by step instruction from 0 to 10 where a kindergardener could understand,That will definetly reduce the amount of people asking for VIDEO's

Then that's not a README but a guide. A project's README should be short, to the point and geared towards developers or tech savvy people. I think your problem is that you're neither of those two meaning you don't understand the most basic concepts nor take the time to research (i.e. google) the things you don't understand...

Anyways, this discussion shouldn't be about any of that. The real issue here is that this project initially started out as something to be setup by people who know what they are doing for use in their private communities. But since the public bot (now demo) gained such traction from noobs/kids/non-tech-savvy-people those kind of people expect self hosting to be as easy as clicking a button (spoiler alert: it's not that easy). You're right that there's not enough materials to get those kind of people started.

I'll try to push out some basic unvoiced vid and/or docs somewhere this week just so you guys can get started and hopefully stop whining about it :)

**Reply by Fishhyyy** [reply]
Thanks

**Comment by sleeyax** [comment]
All video's I wanted to make around the topic are online now, see the playlist here: https://www.youtube.com/playlist?list=PL_QwTUg00Ln_xmpzoGFI-z7_Nj_ZdO0Vr
